src/react-apps/ux-editor/src/containers/Container.tsx

```diff
@@ -168,7 +168,7 @@ export class ContainerComponent extends React.Component<IContainerProps> {
     FormDesignerActionDispatchers.addFormContainer(container, this.props.id);
   }
   public changeActiveFormContainer = () => {
-    if (!this.props.baseContainer) {
+    if (!this.props.baseContainer && this.props.designMode) {
```

Comment:
Really do not know why this is in here. It's a commit from a branch that i have merged with master.